component/admin/controllers/order_detail.php

```diff
@@ -359,7 +359,7 @@ public function send_downloadmail()
                 $msg
             );
         } else {
-            $this->setRedirect('index.php?option=com_redshop&view=order_detail&cid[]=' . $cid[0], $msg, 'error');
+            $this->setRedirect('index.php?option=com_redshop&view=order_detail&cid[]=' . $cid[0], $msg);
```

Comment:
@NguyenBao10 : As exist code, there will be send that email sent failure. If you remove the context `error` It's while not good for the notice message: You should try with `enqueueMessage` that better:
`$app = \JFactory::getApplication();

$app->enqueueMessage($msg, 'error');
$app->redirect(JRoute::_(''index.php?option=com_redshop&view=order_detail&cid[]=' . $cid[0]));`